src/services/ramp-hooks.js

```diff
@@ -652,7 +659,7 @@ export const useActiveStructure = ({
     // Do not mark root range as active
     if (isCurrentSection && !isRoot) {
       // Collapse the section in structured navigation
-      setIsOpen(true);
+      setSectionIsCollapsed(false);
```

Comment:
This comment is confusing given the next line appears to mark the section as not collapsed.  Which is correct?

Reply:
True. I'll change the comment. It should be section is expanded by setting isCollapsed to false.